### PR TITLE
Add "Project partners" and "Project sponsor" to "Projects" page

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/7360

I added project partner and project sponsor the Projects page, as well as the advanced search, the more straightforward search and the csv export configuration. 

## Testing
**URL to test:** 
There are DB changes, so run locally or use moped-test.

**Steps to test:**

From a product summary page, add project partners and/or a project sponsor to a project. They should be visible on the projects page and come up in the search results. 
Screen Shot 2021-12-08 at 1 19 10 PM

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)

### CODE REVIEW

File: moped-database/migrations/1638905763818_run_sql_migration/up.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    mp.updated_at,
+    me.entity_name as project_sponsor,
+    string_agg(me2.entity_name, ', ') as project_partner
+   FROM (((moped_project mp
+     LEFT JOIN moped_proj_personnel mpp ON (((mp.project_id = mpp.project_id) AND (mpp.status_id = 1))))
+     LEFT JOIN moped_users mu ON (mpp.user_id = mu.user_id))
+     LEFT JOIN moped_project_roles mpr ON (mpp.role_id = mpr.project_role_id)
+     LEFT JOIN moped_entity me ON (me.entity_id = mp.project_sponsor))
+     LEFT JOIN moped_proj_partners mpp2 ON ((mp.project_id = mpp2.project_id) and (mpp2.status_id =1))
+     LEFT JOIN moped_entity me2 ON (mpp2.entity_id = me2.entity_id)
```
Comment (chiaberry) [inline moped-database/migrations/1638905763818_run_sql_migration/up.sql:34]: This is what I added.

**Review by sergiogcx — APPROVED** [review]
Looks great! 🙌

**Review by amenity — APPROVED** [review]
LSTM! ✨🚢

**Comment by amenity** [comment]
Oops, missed this... But if you could add an "s" so the column is labeled "Project partners" that would be great, @chiaberry .

**Pull request merged**